test/types/check-output-result.ts

```diff
@@ -0,0 +1,7 @@
+export interface CheckOutputResult<T> {
+  errors: any[]
+  logs: any[]
```

Comment:
could we be more specific with this type, I think these are `string` arrays?

Reply:
Well, we use them as `string[]`, but their true type for now is `any[]`, as we get their contents directly from `console.{error,log,warn}` first parameter `message`, which is `any`.

So if we want to have real `string[]` (and we should, as we always test them again `string[]` afterwards), we should explicitly transform `message` to `string`, using `util.format` for example.

Although this has nothing to do with tsification, I will add a commit to fix that.